### PR TITLE
Build lib and python

### PR DESCRIPTION
**Pull request by wolfv**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by davidbrochart** [comment]
Thanks a log @wolfv!
I don't understand why there is nothing to test for the `chrono` output:
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=125853&view=logs&j=517fe804-fa30-5dc2-1413-330699242c05&t=c7d0fcb4-3a10-5b26-dfea-e879a93fc2ab&l=5489

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,16 +26,43 @@ requirements:
...
-  imports:
-     - pychrono
+outputs:
+  - name: chrono
+    script: clean_python.sh
+    test:
+      commands:
+        - test -f ${PREFIX}/lib/libChronoEngine.so  # [linux]
+        - test -f ${PREFIX}/lib/libChronoEngine.dylib  # [osx]
+
```
Comment (davidbrochart) [inline recipe/meta.yaml]:
```suggestion
        - test -f ${PREFIX}/lib/libChronoEngine.${SHLIB_EXT}
```

**Review by davidbrochart — COMMENTED** [review]
The test doesn't seem to be execute for the `chrono` output, but I checked locally that the library is present.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -37,9 +37,7 @@ outputs:
     script: clean_python.sh
     test:
       commands:
-        - test -f ${PREFIX}/lib/libChronoEngine.so  # [linux]
-        - test -f ${PREFIX}/lib/libChronoEngine.dylib  # [osx]
-
+        - test -f ${PREFIX}/lib/libChronoEngine.${SHLIB_EXT}
```
Comment (wolfv) [inline recipe/meta.yaml:40]: I think ${SHLIB_EXT} evaluates to `.so` etc. so the dot needs to go away :)

File: recipe/meta.yaml [hunk 3]
```diff
@@ -24,16 +26,41 @@ requirements:
...
 
-test:
-  imports:
-     - pychrono
+outputs:
+  - name: chrono
+    script: clean_python.sh
+    test:
+      commands:
+        - test -f ${PREFIX}/lib/libChronoEngine.${SHLIB_EXT}
```
Comment (davidbrochart) [inline recipe/meta.yaml:40]:
```suggestion
        - test -f ${PREFIX}/lib/libChronoEngine${SHLIB_EXT}
```

File: recipe/meta.yaml (hunk 2 above)
Comment (davidbrochart) [reply recipe/meta.yaml:40]: Thanks, I pushed the fix.

**Pull request merged**